OSDOCS-5158 release notes and known issue (#59822)
openshift/openshift-docs: release_notes/ocp-4-13-release-notes.adoc

==== Installing a cluster on GCP using Shielded VMs
In {product-title} {product-version}, you can use Shielded VMs when installing your cluster. Shielded VMs have extra security features including secure boot, firmware and integrity monitoring, and rootkit detection. For more information, see xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installation-gcp-enabling-shielded-vms_installing-gcp-customizations[Enabling Shielded VMs] and Google's documentation on link:https://cloud.google.com/shielded-vm[Shielded VMs].

[id="ocp-4-13-installation-gcp-confidential-vms"]
==== Installing a cluster on GCP using Confidential VMs

> In {product-title} {product-version}, you can use Confidential VMs when installing your cluster. Confidential VMs encrypt data while it is being processed. For more information, see Google's documentation on link:https://cloud.google.com/confidential-computing[Confidential Computing]. You can enable Confidential VMs and Shielded VMs at the same time, although they are not dependent on each other.

Author: Good catch, thanks

[IMPORTANT]
====

> Due to a known issue, you cannot create persistent volume storage on a cluster that uses Confidential VMs. For more information, see link:https://issues.redhat.com/browse/OCPBUGS-7582[OCPBUGS-7582].

Reviewer: ```suggestion
Due to a known issue, you cannot use persistent volume storage on a cluster with Confidential VMs. For more information, see link:https://issues.redhat.com/browse/OCPBUGS-7582[OCPBUGS-7582].
```

* For {product-title} {product-version}, the Driver Toolkit (DTK) container image requires the `ubi9` image as the second layer of the software stack for building driver containers. If you attempt to use the `ubi8` image as the second layer in the your software stack, you will receive a build error. (link:https://issues.redhat.com/browse/OCPBUGS-11120[*OCPBUGS-11120*])

> * Due to a known issue in RHEL 9.2, you cannot create persistent volumes on a GCP cluster that uses Confidential VMs. (link:https://issues.redhat.com/browse/OCPBUGS-7582[*OCPBUGS-7582*])

Reviewer: ```suggestion
* Due to a known issue in RHEL 9.2, you cannot use persistent volumes on a GCP cluster with Confidential VMs. (link:https://issues.redhat.com/browse/OCPBUGS-7582[*OCPBUGS-7582*])
```
